build/azure-devops/scraper-ci.yml

```
@@ -77,6 +77,20 @@ stages:
          solutionOrProjectPath: 'src/Promitor.sln'
          failBuildLevelSelector: 'Warning'
          commandLineInterfacePath: '$(Build.SourcesDirectory)/Lib/Resharper'
+- stage: Linting
+  displayName: Linting
+  dependsOn: []
+  jobs:
+   - job: RunLinters
+     displayName: Run Linters
```

Given we only have this linter it's maybe better to rename this?
```suggestion
     displayName: Run Markdown Linter
```

---

No problem.